src/lib.rs

```diff
@@ -272,8 +283,69 @@ impl Scope {
         }
     }
 
+    /// Returns a mutable reference to a module if it is exists in this scope. 
+    pub fn get_module_mut<Q: ?Sized>(&mut self, 
+                                     name: &Q) 
+                                     -> Option<&mut Module> 
+    where
+        String: PartialEq<Q>,
+    {
+        self.items.iter_mut()
+            .find(|item| if let &mut Item::Module(ref module) = *item { 
```

Comment:
The `find(...).map(...)` sequence could probably be replaced by `filter_map(...).next()` to avoid having to match twice.

Reply:
I thought about that when implementing this originally, and then forgot. Changed in b40b1e61ec1bbc6459924cb08e6193bd870b39bf.